service/src/LevelZero.cpp

```diff
@@ -552,6 +609,74 @@ namespace geopm
         return result;
     }
 
+    int LevelZeroImp::ras_domain_count(unsigned int l0_device_idx,
+                                       int l0_domain) const
+    {
+        return m_devices.at(l0_device_idx).subdevice.ras_domain.at(l0_domain).size();
+    }
+
+    // TODO RAS check return datatype - uint64 to double
+    double LevelZeroImp::ras_reset_count(unsigned int l0_device_idx,
+                                         int l0_domain, int l0_domain_idx) const
+    {
+        return (ras_status_helper(l0_device_idx, l0_domain, l0_domain_idx))[ZES_RAS_ERROR_CAT_PROGRAMMING_ERRORS];
```

Comment:
```suggestion
        return (ras_status_helper(l0_device_idx, l0_domain, l0_domain_idx))[ZES_RAS_ERROR_CAT_RESET];
```
This was wrong.  Please double check all of these.

Reply:
Thanks. Fixed.